### PR TITLE
add column mapping check

### PR DESCRIPTION
**Pull request by IANTHEREAL**

### CODE REVIEW

File: pkg/check/table_structure.go [hunk 1]
```diff
@@ -327,17 +337,52 @@ func (c *ShardingTablesCheck) Check(ctx context.Context) *Result {
...
-		r.Instruction = "please set column mapping rules for them, or handle it by yourself"
-		r.Extra = AutoIncrementKeyChecking
+	return r
+}
+
+func (c *ShardingTablesCheck) checkAutoIncrementKey(instance, schema, table string, ctStmt *ast.CreateTableStmt, info *model.TableInfo, r *Result) bool {
+	autoIncrementKeys := c.findAutoIncrementKey(ctStmt, info)
+	for columnName, isBigInt := range autoIncrementKeys {
+		hasMatchedRule := false
+		if cm, ok1 := c.mapping[instance]; ok1 {
```
Comment (csuzhangxc) [inline pkg/check/table_structure.go:347]: now, if `autoIncrementKeys` found (with multi sharding tables), but no `mapping` set, we will not output warning?

Comment (csuzhangxc) [reply pkg/check/table_structure.go:347]: maybe you want to move L369 ~ L375 out from `if` (L347) block?

Comment (IANTHEREAL) [reply pkg/check/table_structure.go:347]: it would output error

File: sync_diff_inspector/diff.go [hunk 2]
```diff
@@ -195,7 +195,7 @@ func (df *Diff) AdjustTableConfig(cfg *Config) error {
 
 			allTables, ok := allTablesMap[schemaStr(df.sourceDBs[sourceTable.InstanceID].InstanceID, sourceTable.Schema)]
 			if !ok {
-				return errors.Errorf("unknow schema %s in database %s", sourceTable.Schema, df.sourceDBs[sourceTable.InstanceID])
+				return errors.Errorf("unknow schema %s in database %+v", sourceTable.Schema, df.sourceDBs[sourceTable.InstanceID])
```
Comment (kennytm) [inline sync_diff_inspector/diff.go]: btw, "unknown" schema

**Comment by IANTHEREAL** [comment]
@kennytm @csuzhangxc PTAL

**Comment by kennytm** [comment]
LGTM

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Pull request merged**